laalaa/apps/advisers/healthchecks.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from moj_irat.healthchecks import HealthcheckResponse, UrlHealthcheck, registry
+
+class CeleryWorkersHealthcheck(object):
+    def __init__(self, name):
+        self.name = name
+    
+    def __call__(self, *args, **kwargs):
+        from celery import Celery
+        
+        try:
+            app = Celery('laalaa')
+            stats = app.control.inspect().stats()
+            if not stats:
+                return self.error_response('No running workers were found.')
+            workers = stats.values()
+            if len(workers) == 0:
+                return self.error_response('No workers running')
```

Comment:
_consistency nitpick_
The rest of the sentences have full stops at the end :neckbeard:

Reply:
Fixed